schema/deploy/shipping/views.sql

```diff
@@ -844,4 +844,73 @@ comment on view shipping.hcov19_observation_v1 is
   'Custom view of hCoV-19 samples with presence-absence results and best available encounter data';
 
 
+create or replace view shipping.scan_return_results_v1 as
+    select distinct on (barcode, lineage)
+        sample_id,
+        barcode as qrcode,
+        encountered as collect_ts,
```

Comment:
I'd make this a date instead of a timestamp. The time may give a false sense of precision and is not necessary for returning results.